source/LumiCalReco/src/LumiCalClusterer_auxiliary.cpp

```diff
@@ -133,8 +134,10 @@ int LumiCalClustererClass::getNeighborId(int cellId, int neighborIndex) {
   GlobalMethodsClass::CellIdZPR( cellId, cellZ, cellPhi, cellR, arm);
 
   // change iRho cell index  according to the neighborIndex
-  if ( neighborIndex%2 ) cellR -= neighborIndex+1;
-  else cellR += neighborIndex+1;
+  if( neighborIndex >= 2 ){
+    if ( neighborIndex%2 ) cellR -= neighborIndex/2;
+    else cellR += neighborIndex/2;
+  }
 
```

Comment:
Yes right, but lines 137-140 are equivalent to one like this:
cellR = ( neighbourIndex%2 ) ? cellR -= neighbourIndex/2 : cellR += neighbourIndex/2 ;